mattermost.py

```diff
@@ -88,13 +100,17 @@ def make_data(args):
 
 
 def request(url, data):
-    req = urllib2.Request(url, data)
-    response = urllib2.urlopen(req)
+    if IS_PY2:
```

Comment:
You can use urllib in both Python and Python2.7. Therefore this verification is unnecessary.